agent/src/main/java/io/pyroscope/javaagent/JFRJDKProfilerDelegate.java

```diff
@@ -53,11 +54,24 @@ private static File jfrRecordingPath() throws IOException {
     public synchronized void start() {
         try {
             recording = new Recording();
-            recording.enable("jdk.ExecutionSample").withPeriod(Duration.ofMillis(1));
-            recording.enable("jdk.ThreadPark").withPeriod(Duration.ofMillis(10)).withStackTrace();
-            recording.enable("jdk.ObjectAllocationInNewTLAB").withStackTrace();
-            recording.enable("jdk.ObjectAllocationOutsideTLAB").withStackTrace();
-            recording.enable("jdk.JavaMonitorEnter").withPeriod(Duration.ofMillis(10)).withStackTrace();
+            switch (config.profilingEvent) {
+                case CPU: {
+                    recording.enable("jdk.ExecutionSample").withPeriod(Duration.ofMillis(1));
+                    break;
+                }
+                case ALLOC: {
+                    recording.enable("jdk.ObjectAllocationInNewTLAB").withStackTrace();
```

Comment:
Any chance we can use Configuration to somehow configure these events? in asprof we have `alloc=256k`, can we somehow configure it here as well?

Reply:
nope, unfortunately this settings is not exposed at the moment through recording object :(